src/plugins/mf_priority.cpp

```diff
@@ -368,9 +368,10 @@ static int check_and_release_held_jobs (flux_plugin_t *p, Association *b)
                                  held_job_id,
                                  "mf_priority",
                                  0,
-                                 "job.state.inactive: failed to remove %s "
-                                 "dependency from job",
-                                 dependency.c_str ());
+                                 "check_and_release_held_jobs: failed to "
+                                 "remove %s dependency from job %lld",
```

Comment:
Suggestion: use `%ju` for `flux_jobid_t` format spec to avoid displaying negative numbers.

In the future you could look into pulling in `idf58.h` from flux-core's `src/common/libjob` which allows f58 jobids to easily be printed.